src/utils/framediff_utils.py

```diff
@@ -0,0 +1,93 @@
+import cv2
+import numpy as np
+
+"""
+
+Utility functions to calculate inter-frame difference
+
+"""
+
+class DistanceMetrics:
```

Comment:
We normally don't implement classes in utils. I mean util functions are anyways meant to be static methods so why keep them in a class when we are not using any object state.